### ISSUE TITLE
Use gps precision when computing hexagon

### ISSUE THREAD

**Issue by nicolas-f** [Ifsttar/NoiseCapture#167]
Include  measure point in hexagons intersecting with the circle of the measure point defined by it's precision.

**Comment by ebocher** [comment]
Don't understand.
Do you want to collect the data for each hexa according the GPS precision ?

**Comment by Picaut** [comment]
The idea is to affect the noise level value of a given point to the corresponding hexagons in function of the value of the GPS accuracy for the corresponting point. For example, if the GPS accuracy is 20m for a given point, then the sound level at this point could be affected to several hexagons.

**Comment by pierromond** [comment]
But it means that we could affect several times the same value ? Also, worst will be the GPS accuracy, more weight will have the value ?

**Comment by Picaut** [comment]
Yes. The point could be on each on the hexagon. It seems difficult to weight the sound level at this step. Remember that this first map (with the hexagons) is a "raw" map including all raw data without any filtering. We will be able to propose new maps (with a better process) in a next step.
